lib/custom-events/user-events.js

```diff
@@ -0,0 +1,6 @@
+const EventEmitter = require('events');
+
+class UserEvents extends EventEmitter { };
```

Comment:
Retain original nomenclature to indicate appropriate class name.
```suggestion
class AuthXEventEmitter extends EventEmitter { };
```